File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.cc
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main() {
    string s;
    cin >> s;

    vector<int> iArray(26, -1);
    for (int i = 0; i < s.length(); i++) {
        char cChar = s[i];

        if (iArray[cChar - 'a'] == -1) {
            iArray[cChar - 'a'] = i;
        }
    }
    for (int i = 0; i < 26; i++) {
        cout << iArray[i] << " ";
    }

    return 0;
}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String s = br.readLine();

		int iArray[] = new int[26];
		for (int i = 0; i < 26; i++) {
			iArray[i] = -1;
		}		
		for (int i = 0; i < s.length(); i++) {
			char cChar = s.charAt(i);

			if (iArray[cChar - 97] == -1) {
				iArray[cChar - 97] = i;
			}
		}
		for (int i = 0; i < 26; i++) {
			System.out.print(iArray[i] + " ");
		}
	}
}